django/cantusdb_project/main_app/models/chant.py

```diff
@@ -148,3 +149,62 @@ def get_next_folio(folio):
             next_chant = None
 
         return next_chant
+
+    def get_syllabized_text_and_melody(self):
+        """Returns the syllabized text with the melody of a chant. If volpiano is not established return None
```

Comment:
Maybe say here what we mean by "syllabized text"? An short example?

Reply:
Added more description to the docstring